cfpb/cfgov-refresh
# Upgrade cfpb-chart-builder to v2.0.2, add highcharts css

Latest version of `cfpb-chart-builder`. Fixes the way that the default highcharts css is imported into the project css. 

The previous version caused a significant performance degradation because I had mistakenly left a statement importing it as an http request in our less file. This new version imports it from local npm dependency. See PR for this release over here: https://github.com/cfpb/cfpb-chart-builder/pull/65

Here's an example of the bug, if you open up Network tab in Chrome Dev Tools you will see a separate request for 'highcharts.css' that we don't need to make: https://beta.consumerfinance.gov/data-research/consumer-credit-trends/auto-loans/origination-activity/

## Additions

- Added highcharts path to the gulp config so that we can import the css from local `node_modules` directory when building front end.


## Testing

1. checkout branch
1. `source .env`
1. `./setup.sh`
1. `./runserver.sh`
1. Open Dev Tools Network tab
1. visit http://localhost:8000/data-research/consumer-credit-trends/auto-loans/origination-activity/
1. Confirm graphs look as expected
1. Confirm there is no `highcharts.css` file loaded in your list of requested assets (click 'CSS' to see all css files loaded by page)

## Screenshots

- No highcharts.css in here
![screen shot 2017-07-28 at 11 41 12 am](https://user-images.githubusercontent.com/702526/28724932-d19b4ba2-7389-11e7-9b3c-2e5d7ba0b56b.png)



## Checklist
